web/src/components/CreateTestPlugins/Kafka/steps/RequestDetails/RequestDetailsMessageKeyInput.tsx

```diff
@@ -0,0 +1,17 @@
+import {Form} from 'antd';
+import {SupportedEditors} from 'constants/Editor.constants';
+import Editor from 'components/Editor';
+import * as S from './RequestDetails.styled';
+
+const RequestDetailsMessageKey = () => {
+  return (
+    <div>
```

Comment:
Instead of using `div` you can use fragments `<></>`

Reply:
Makes sense, but here since each value is on its own row, if I remove the `div`, I get a weird margin spacing. Here is an example with of them without `div`:
<img width="514" alt="image" src="https://github.com/kubeshop/tracetest/assets/12397879/b5fc9498-f062-45ba-a19b-c7a0a7fd0790">